### PR TITLE
Grafana add configuration dashboard and datasource

### PR DESCRIPTION
**Pull request by Tezno**

### CODE REVIEW

File: src/cmd/api/router/router.go [hunk 1]
```diff
@@ -39,3 +49,38 @@ func getCORS() (handlers.CORSOption, handlers.CORSOption, handlers.CORSOption, h
 
 	return corsObj, methods, headers, credentials
 }
+
+// Metrics
```
Comment (RaboliotLeGris) [inline src/cmd/api/router/router.go:53]: I'm not sure to understand what is done with the middleware and the NewResponseWrite, can you add a bit of context/ some comments ?

File: src/cmd/api/router/router.go [hunk 2]
```diff
@@ -2,25 +2,35 @@ package router
...
 	"github.com/gorilla/mux"
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	log "github.com/sirupsen/logrus"
 
 	"github.com/Sogilis/Voogle/src/pkg/clients"
 
 	"github.com/Sogilis/Voogle/src/cmd/api/config"
 	"github.com/Sogilis/Voogle/src/cmd/api/controllers"
+	metrics "github.com/Sogilis/Voogle/src/cmd/api/metrics"
```
Comment (RaboliotLeGris) [inline src/cmd/api/router/router.go:18]: As I said in the previous (RIP) PR, is the alias necessary ?

File: src/cmd/api/router/router.go [hunk 3]
```diff
@@ -39,3 +49,38 @@ func getCORS() (handlers.CORSOption, handlers.CORSOption, handlers.CORSOption, h
 
 	return corsObj, methods, headers, credentials
 }
+
+// Metrics
+func initMetrics() {
```
Comment (RaboliotLeGris) [inline src/cmd/api/router/router.go:54]: Maybe we can move this part where the metrics are defined ?

File: docker/docker-compose-observability.yml [hunk 4]
```diff
@@ -1,13 +1,17 @@
...
+  grafana_data: {}
   prometheus_data: {}
 
 services:
   grafana:
     image: grafana/grafana:8.4.3@sha256:44ea09598b05bd6296f663bb728eb6af90f45399648d85b4123bbc3f271a121b
     container_name: grafana
     restart: unless-stopped
+    volumes: 
+      - ./.grafana/provisioning/:/etc/grafana/provisioning/
```
Comment (RaboliotLeGris) [inline docker/docker-compose-observability.yml:13]: Maybe we can store all the tempory data in a path like : docker/.data/{service}

Comment (Tezno) [reply docker/docker-compose-observability.yml:13]: Solution is https://github.com/cfbarbero/tick-grafana-docker/issues/1#issuecomment-636894425. So I decide to keep docker volumes

**Review by RaboliotLeGris — APPROVED** [review]

**Pull request merged**